sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace-with-jsdom-unit-testing_sjs-0.6/build.sbt

```diff
@@ -3,7 +3,7 @@ name := "global-namespace-with-jsdom-unit-testing"
 
 enablePlugins(ScalaJSBundlerPlugin, ScalaJSJUnitPlugin)
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.11"
```

Comment:
Why not 2.13.2 here?

Reply:
The facade is not available for 2.13, this I kept 2.12